command/base.go

```diff
@@ -275,27 +284,8 @@ func (c *BaseCommand) validateMFA(reqID string, methodInfo MFAMethodInfo) int {
 		"mfa_request_id": reqID,
 		"mfa_payload":    mfaPayload,
 	})
-	if err != nil {
-		c.UI.Error(err.Error())
-		if secret != nil {
-			OutputSecret(c.UI, secret)
-		}
-		return 2
-	}
-	if secret == nil {
-		// Don't output anything unless using the "table" format
-		if Format(c.UI) == "table" {
-			c.UI.Info(fmt.Sprintf("Success! Data written to: %s", path))
-		}
-		return 0
-	}
-
-	// Handle single field output
-	if c.flagField != "" {
-		return PrintRawField(c.UI, secret, c.flagField)
-	}
 
-	return OutputSecret(c.UI, secret)
```

Comment:
Why does this differ from the original PR?

Reply:
I think the two main differences are:
- 1.10.x doesn't have the `Client.MFAValidate()` api
- 1.10.x doesn't have `WithContext` api calls